java/src/test/resources/mixed_types_1.json

```diff
@@ -0,0 +1,2 @@
+{ "a": "123" }
```

Comment:
Can these test cases be defined inline rather than as separate files? If it can be read from a string in memory, that would be ideal. If the test would require writing the file to disk in order to read it, then it's okay to keep this as a file in the repo.

Reply:
The files are present to test difference sources.